gpjax/kernels/computations/base.py

```diff
@@ -15,15 +15,15 @@
 
 import abc
 from dataclasses import dataclass
-from typing import Any
+from beartype.typing import Any
 
 from jax import vmap
 from jaxtyping import Array, Float
 
 from gpjax.linops import (DenseLinearOperator, DiagonalLinearOperator,
                           LinearOperator)
 
-Kernel = Any
+Kernel = Any  # TODO: FIX
```

Comment:
this doesn't quite seem right. should we just use a string forward reference instead?

Reply:
Yes.

Reply:
done